spec/system/feedback_spec.rb

```diff
@@ -23,10 +23,11 @@
         click_on "Continue to download and finish"
         expect(page).to have_content("Were you satisfied with this service?")
         click_on "Yes"
-        expect(page).to have_content("Your response has been sent, please tell us more")
+        assert page.has_content?("Your response has been sent, please tell us more")
+        assert page.has_content?("Don't include personal information")
```

Comment:
This seems a curious change in style? expect(x).to have_content() should work fine

Reply:
Thanks @starswan for the spot. This is me trying out some things, to stop the race between browser and server on this spec. I forgot to take them out when the didn't have much of an effect. I think I was following [this blog](https://nts.strzibny.name/avoid-sleep-rails-system-tests/). 

As an aside, there is this thing called [rspec-wait](https://github.com/laserlemon/rspec-wait), which we could use?

Reply:
It's idiomatic to call find(<css-selector>) that waits a default amount of time before failing